src/Compilers/CSharp/Test/Semantic/Semantics/FunctionPointerTests.cs

```diff
@@ -618,11 +622,16 @@ public void FunctionPointerToFunctionPointerInvalid_ParameterVariance()
             var comp = CreateCompilationWithFunctionPointers(@"
 unsafe class C
 {
-    void M(delegate*<ref object, void> param1)
+    void M(delegate*<ref object, void> param1, delegate*<ref object, void> param2)
```

Comment:
>delegate*<ref object, void> [](start = 47, length = 27)

This is the same type as for param1. Why is it useful to add it?

Reply:
Whoops, this should have been an out. Will fix.

---
In reply to: [409822981](https://github.com/dotnet/roslyn/pull/43355#discussion_r409822981) [](ancestors = 409822981)